### PR TITLE
Fix bug that "current position" events are not emitted in Flutter Web

### LINKED ISSUE #985: (with PR included) "current position" events are not emitted in Flutter Web
Hi thanks for the lib! I find that "current position" events are not emitted in Flutter Web. So I make a PR: #984

### PR DESCRIPTION
**Pull request by fzyzcjy**
Closes #985

### CODE REVIEW

File: packages/audioplayers/lib/src/audioplayers_web.dart [hunk 1]
```diff
@@ -44,6 +49,9 @@ class WrappedPlayer {
     player?.loop = shouldLoop();
     player?.volume = currentVolume;
     player?.playbackRate = currentPlaybackRate;
+    playerTimeUpdateSubscription = player?.onTimeUpdate.listen((event) =>
+        plugin.channel.invokeMethod('audio.onCurrentPosition',
+            {'value': (1000 * (player?.currentTime ?? 0)).round()}));
```
Comment (luanpotter) [inline packages/audioplayers/lib/src/audioplayers_web.dart:54]: wrong formatting here

```suggestion
        plugin.channel.invokeMethod(
            'audio.onCurrentPosition',
            {'value': (1000 * (player?.currentTime ?? 0)).round()},
        ),
    );
```

**Review by luanpotter — COMMENTED** [review]
I think it was not a bug, just never implemented. Could you please also add a line to the changelog & update the [feature parity](https://github.com/luanpotter/audioplayers/blob/master/feature_parity_table.md) table to include this development?

**Comment by fzyzcjy** [comment]
@luanpotter sure, done

**Review by luanpotter — APPROVED** [review]
LGTM

**Pull request merged**